ninja-core/src/test/java/conf/Module.java

```diff
@@ -18,13 +18,26 @@
 
 import com.google.inject.AbstractModule;
 
+import ninja.utils.NinjaProperties;
+
 
 // Just a dummy for testing.
 // Allows to check that custom Ninja module in user's conf directory
 // works properly.
 public class Module extends AbstractModule {
 
-  
+	NinjaProperties ninjaProperties;
+	
+	public Module(NinjaProperties ninjaProperties) {
+		
+		// Will fail any test based on this Module if properties not received
+		if(ninjaProperties == null) {
+			throw new IllegalArgumentException("null properties");
```

Comment:
Please use Preconditions from Guava (checkNotNull or similar). And please print out a better error message. Just for the very rare case when something goes wrong this will help a lot to find the problem asap... :)